classes/Guest.php

```diff
@@ -157,6 +157,7 @@ protected function getBrowser($userAgent)
     protected function getOs($userAgent)
     {
         $osArray = [
+            'Windows 11' => 'Windows NT 11',
```

Comment:
```suggestion
            'Windows 11' => 'Windows 11',
```
It's the first version of Windows that doesn't only use the Windows NT kernel, but also a Linux kernel ^^
It's a FYI